aspen/resources/simplate.py

```diff
@@ -231,11 +231,15 @@ def _unbound_parse_specline(self, specline):
         #Assign parts
         media_type, renderer = parts
         if renderer == '':
+            # no renderer specified, use the default
             renderer = self.website.default_renderers_by_media_type[media_type]
+        if media_type == '':
+            # no media type specified, use the default
+            media_type = self.media_type
```

Comment:
Wouldn't it be better to fill `media_type` before `renderer` since the latter depends on the former?